test/functional/lua/ui_event_spec.lua

```diff
@@ -365,6 +365,11 @@ describe('vim.ui_attach', function()
           history = true,
           kind = 'return_prompt',
         },
+        {
+          content = { { "E354: Invalid register name: '^@'", 9, 6 } },
+          history = false,
+          kind = '',
+        },
```

Comment:
These tests are too flake, and I believe incorrect. For example because the input here ends with `<CR>`, I don't believe it should show "Press ENTER". 

In any case, they result is totally dependent on which UI events are sent. With these changes, `win_pos` and `grid_destroy` are no longer sent to non-multigrid UIs, which affects the test results, even if they are not handled.